package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "glints-aries",
-  "version": "4.0.0-beta.66",
+  "version": "4.0.66",
```

Comment:
I had a conversation with @franciscrispin about this a while ago: We kinda agreed to keep the pre-releases for the time being:

```
marco 6:17
What’s up with the version numbers anyway?
6:18
How long is this pre-release thing going to go? ^^

francis  6:19 PM
I was planning to officially release and do away with the pre-releases once testing was completed haha. Don't feel confident releasing it without solid test coverage :sweat_smile:

marco  6:19 PM
At this point, what’s the difference?^^

francis  6:19 PM
As for the version numbers we can just keep incrementing it while it is in beta
6:22
Hmm no difference now that I think about it. I think at that point I still considered it a "public" library so I wanted to wait for the test coverage to be 100% before releasing haha
```

Though I still think it doesn't matter too much. If the pre-release thing is a problem for the script let's switch to regular semvers.